chain-signatures/node/src/protocol/signature.rs

```diff
@@ -227,18 +211,35 @@ pub struct GenerationRequest {
     pub proposer: Participant,
     pub request: ContractSignRequest,
     pub epsilon: Scalar,
-    pub receipt_id: CryptoHash,
+    pub receipt_id: [u8; 32],
     pub entropy: [u8; 32],
     pub sign_request_timestamp: Instant,
 }
 
+#[derive(Debug, Clone, Eq, Hash, PartialEq)]
+pub struct SignRequestIdentifier {
+    pub receipt_id: [u8; 32],
```

Comment:
receipt is still in the name here

Reply:
removed receipt_id field from everywhere but indexer